bedrock/firefox/templates/firefox/features/sync.html

```diff
@@ -34,156 +34,78 @@
 {% endblock %}
 
 {% block features_header_content %}
-  <h1><span>{{ _('Firefox Sync') }}</span></h1>
+<h1>
+  <span class="show-not-fx show-fx-android show-fx-ios show-fx-31-signed-out show-fx-30-older show-default">{{ _('Browse uninterrupted with Firefox Sync') }}</span>
+  <span class="show-fx-31-signed-in">{{ _('Sync across devices') }}</span>
+</h1>
+<p class="tagline show-not-fx show-fx-31-signed-out show-fx-30-older show-default">{{ _('With Firefox on all your devices, you can access bookmarks, tabs and passwords with one easy sign in.') }}</p>
```

Comment:
Pointed out by @peiying2 : this should be "sign-in", not "sign in" (verb).
https://english.stackexchange.com/questions/49850/sign-in-signin-or-sign-in

Are we OK with fixing it without going back to copy?

Reply:
I can fix this, np